torch/_higher_order_ops/utils.py

```diff
@@ -651,3 +651,54 @@ def _tensor_storage(t) -> StorageWeakRef:
             if isinstance(inp, torch.Tensor) and _tensor_storage(inp) in out_storage_map
         }
         return mutated_inputs, inp_inp_alias_map, inp_out_alias_map, out_out_alias_map
+
+
+registered_hop_fake_fns: dict[torch._ops.OpOverload, Callable] = {}
+
+
+def register_hop_fake(hop, fn=None):
```

Comment:
nit: call this just register_fake? the fqn already has HOP in it (torch._higher_order_ops.register_fake)